central/views/imagecve/view_impl.go

```diff
@@ -197,7 +197,9 @@ func withSelectCVECoreResponseQuery(q *v1.Query, cveIDsToFilter []string, option
 	cloned := q.Clone()
 	if len(cveIDsToFilter) > 0 {
 		cloned = search.ConjunctionQuery(cloned, search.NewQueryBuilder().AddDocIDs(cveIDsToFilter...).ProtoQuery())
-		cloned.Pagination = &v1.QueryPagination{SortOptions: q.GetPagination().GetSortOptions()}
+		if q.GetPagination() != nil && q.GetPagination().GetSortOptions() != nil {
```

Comment:
nit:  comment on why we only want the sort options.

Reply:
Rethinking, I think this query change should be done by the caller for `withSelectCVECoreResponseQuery` because caller has the split query logic. I moved it there and added the comment.